tidy3d/components/device_spec.py

```diff
@@ -48,4 +48,29 @@ class SolidSpec(AbstractHeatSpec):
     )
 
 
-HeatSpecType = Union[FluidSpec, SolidSpec]
+class InsulatorSpec(AbstractDeviceSpec):
+    """Insulating medium.
```

Comment:
is it perfectly insulating?  (I dont know much about thermal modeling..)

Reply:
it just tells us to not solve for electric potential in structures with this `electric_spec`. I have updated the docstring for both `InsulatorSpec` and `FluidSpec`